r1cs-std/src/test_constraint_system.rs

```diff
@@ -63,6 +62,7 @@ impl<ConstraintF: Field> TestConstraintSystem<ConstraintF> {
         }
     }
 
+    #[allow(unused_variables)]
```

Comment:
Why is this needed here?

Reply:
a redefinition of `println` on `no-std` doesn't do anything

Reply:
ahh so the idea is that since the no_std version of the macro doesn't do anything with the variable, it leads to the variable being left unused. Ok, makes sense.